janusgraph-core/src/main/java/org/janusgraph/graphdb/olap/job/IndexUpdateJob.java

```diff
@@ -139,6 +140,10 @@ public void workerIterationEnd(ScanMetrics metrics) {
             log.error("Transaction commit threw runtime exception:", e);
             metrics.incrementCustom(FAILED_TX);
             throw e;
+        } finally {
+            log.info("Index {} metrics success-tx {} doc-updates  {} success {}", indexName,
```

Comment:
(nitpick) I would change the format slightly, e.g., like this:
```
"Index {} metrics: success-tx: {} doc-updates: {} succeeded: {}"
```

Reply:
Updated :-)